crypto-onramp/src/main/java/com/stripe/android/crypto/onramp/model/OnrampVerifyKycInfoResult.kt

```diff
@@ -0,0 +1,38 @@
+package com.stripe.android.crypto.onramp.model
+
+import androidx.annotation.RestrictTo
+
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+fun interface OnrampVerifyKycCallback {
+    fun onResult(result: OnrampVerifyKycInfoResult)
+}
+
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+sealed interface OnrampVerifyKycInfoResult {
+    /**
+     * Kyc verified successfully.
```

Comment:
nit: "KYC" here and below in docstrings 
```suggestion
     * KYC verified successfully.
```